server/updateStore.ts

```diff
@@ -0,0 +1,23 @@
+"use server";
+
+import prisma from "@/lib/prisma";
+import { Store } from "@prisma/client";
+
+export async function updateStore(
+  store: Store,
+  newDisplayName: string,
+  newFullName: string,
+  newSuburb: string
+) {
+  await prisma.store.update({
+    where: {
+      displayName: store.displayName,
+      suburb: store.suburb,
+    },
```

Comment:
this will update all stores that match the `displayName` + `suburb`. Although we block anything having the same display name and suburb right now, we might allow that in the future (Think many users, each having their own set of stores - they should both be allowed to add ALDI Richmond but stored as separate entries so we can allow them to update the one they added but not one that they didn't add)

For this reason we always need to specify the record that we are modifying by its `id`, which is guaranteed to be unique

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/21/commits/a4ec86cd28f5ee19f1e947bd5940c39e3a4cf6fd